lib/mincer/asset_attributes.js

```diff
@@ -150,9 +150,9 @@ getter(AssetAttributes.prototype, 'extensions', function () {
  *      // -> ".js"
  **/
 getter(AssetAttributes.prototype, 'formatExtension', function () {
-  return _.find(this.extensions.reverse(), function (ext) {
+  return _.find(this.extensions.reverse(), _.bind(function (ext) {
```

Comment:
What are reasons to use lodash's `_.bind()` instead of native `.bind()`?

Reply:
The reason for that, is because I followed the details in the Lodash change log, for Lodash 3 to 4 compatibility. which suggests this way https://github.com/lodash/lodash/wiki/Changelog#compatibility-warnings

Reply:
`_.bind()` cares about special cases like `Object`, but we have only functions here. I thing native `.bind()` will be ok, and less messy.

Reply:
OK cool, will update